CRM/Event/Form/ManageEvent/TabHeader.php

```diff
@@ -66,7 +66,14 @@ public static function process(&$form) {
     $tabs = [];
     $tabs['settings'] = ['title' => ts('Info and Settings'), 'class' => 'ajaxForm livePage'] + $default;
     $tabs['location'] = ['title' => ts('Event Location')] + $default;
-    $tabs['fee'] = ['title' => ts('Fees')] + $default;
+    // Check to see if CiviContribute is an enabled component.
+    $components = \Civi\Api4\Setting::get()
+      ->addSelect('enable_components')
+      ->execute()[0]['value'];
+    // If CiviContribute is active, create the Fees tab.
+    if (in_array('CiviContribute', $components)) {
```

Comment:
Actually - even pithier....
`if (CRM_Core_Component::isEnabled('CiviContribute')) {`

Reply:
Thank you @eileenmcnaughton - that is *much* pithier! I've modified the code to use that helper instead of the API call to achieve the conditional logic.